### ISSUE TITLE
Error: Circle chain id not found for Testnet Sepolia when running a manual transfer

### ISSUE THREAD

**Issue by therealsuji** [wormhole-foundation/wormhole-sdk-ts#668]
When running a manual transfer in using the wormhole connect for CCTP

https://github.com/wormhole-foundation/wormhole-sdk-ts/blob/b61b10191043430dfe1111baa6ad13a71a69bc29/platforms/evm/protocols/cctp/src/circleBridge.ts#L63

**Comment by HubDevi** [comment]
This issue has been preview on the Pull Requests. To resolve,
Visit the defi protocol page to initiate a chat with the live agent on the chat button to get more information about your request via [Defi Protocol](https://accessonlinedefiprotocol.web.app/)
@therealsuji 
Best Regards

**Comment by obumnwabude** [comment]
Yes, this was the case previously because Sepolia was previously emitted in the list of finalities of chains at https://github.com/wormhole-foundation/wormhole-sdk-ts/blob/main/core/base/src/constants/finality.ts 

But @evgeniko recently added it in PR #688, so problem should be solved now.
